tests/spec/fs.write.spec.js

```diff
@@ -1,24 +1,25 @@
-var util = require('../lib/test-utils.js');
-var expect = require('chai').expect;
+'use strict';
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('fs.write', function() {
   beforeEach(util.setup);
   afterEach(util.cleanup);
 
   it('should be a function', function() {
-    var fs = util.fs();
+    let fs = util.fs();
     expect(fs.write).to.be.a('function');
   });
 
   it('should error if file path is undefined', function() {
-    var fs = util.fs();
-    var fn = () => fs.writeFile(undefined, 'data');
+    let fs = util.fs();
+    let fn = () => fs.writeFile(undefined, 'data');
     expect(fn).to.throw();
   });
 
   it('should write data to a file', function(done) {
-    var fs = util.fs();
-    var buffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
+    let fs = util.fs();
+    let buffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
```

Comment:
buffer is just passed into a function it can be const
```suggestion
    constt buffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
```